packages/Webkul/Shop/src/Http/Resources/ProductReviewResource.php

```diff
@@ -14,12 +14,19 @@ class ProductReviewResource extends JsonResource
      */
     public function toArray($request)
     {
+        $images = $this->images->map(function ($image) {
+            $image->url = $image->url;
+        
+            return $image;
+        });
```

Comment:
Remove this logic. And add the `url` key to the appends property.

Reply:
done